api/src/routes/publications.js

```diff
@@ -17,6 +17,8 @@ publicationsRouter.post('/', publicationsMiddleware.createPublicationValidation,
 
 publicationsRouter.get('/:id', publicationsController.readPublication);
 
+publicationsRouter.get("/import/:id", publicationsMiddleware.validateAuthorId, publicationsController.getGoogleScholarPublications);
```

Comment:
Is the `:id` parameter clear enough in this context? I'm thinking it might be ambiguous what id it's referring to. Maybe a more descriptive param? e.g. `:gScholarUserId` or similar.